.github/workflows/issue_assign.yml

```
@@ -0,0 +1,20 @@
+name: Issue Assign Command
+on:
+  issue_comment:
+    types: created
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
+
+jobs:
+  issue_assign:
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-latest
```
Should we run this on latest? I'm not well versed with github actions yet, but seems like a good idea. What do you think?

---

We can go with latest sure, makes no difference for this action.
I will updated.